schema/cassandra.cql

```diff
@@ -0,0 +1,38 @@
+DROP KEYSPACE IF EXISTS gallery_service;
+
+CREATE KEYSPACE gallery_service WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 } AND durable_writes = 'true';
+
+USE gallery_service;
+
+CREATE TABLE IF NOT EXISTS listings (
```

Comment:
if you are using listing ID to find the listing, you should make it clear with the table naming.